setup.py

```diff
@@ -97,29 +118,9 @@ def read(*rnames):
       package_dir={'': 'src'},
       namespace_packages=['zope'],
       ext_modules=ext_modules,
-      extras_require={
-          'docs': [
-              'Sphinx',
-              'repoze.sphinx.autointerface',
-          ],
-          'test': [
-              'zope.testing',
-              'zope.testrunner',
-          ],
-          'zcml': [
-              'zope.component[zcml]',
-              'zope.configuration',
-              'zope.security[zcml]>=4.0.0a3',
-          ],
-          'zodb': [
-              'ZODB>=3.10',
-          ],
-      },
+      extras_require=extras,
       install_requires=install_requires,
-      tests_require=[
-          'zope.testing',
-          'zope.testrunner',
-      ],
+      tests_require=extras['test'],
```

Comment:
Weren't we dropping `setup.py test` everywhere?  test_require is not useful for anything else.